src/net.h

```diff
@@ -418,6 +426,29 @@ class CConnman
     std::atomic<NodeId> nLastNodeId{0};
     unsigned int nPrevNodeCount{0};
 
+    /**
+     * Cache responses to addr requests to minimize privacy leak.
+     * Attack example: scraping addrs in real-time may allow an attacker
+     * to infer new connections of the victim by detecting new records
+     * with fresh timestamps (per self-announcement).
+     */
+    struct CachedAddrResponse {
+        std::vector<CAddress> m_addrs_response_cache;
```

Comment:
acd6135b43941fa51d52f5fcdb2ce944280ad01e

nit, no need for `m_` prefix in structs (?).

Reply:
I assumed that all member variables (of classes and structs) should have the `m_` prefix.

Reply:
Yeah Ive got the same impression.

Reply:
See some recently added structs. Nit anyway.

Reply:
I see `TxDownloadState` has `m_` prefix :)